config/manifests/kustomization.yaml

```diff
@@ -0,0 +1,59 @@
+---
+# yamllint disable rule:line-length
+# These resources constitute the fully configured set of manifests
+# used to generate the 'manifests/' directory in a bundle.
+resources:
+  - bases/freeipa-operator.clusterserviceversion.yaml
+  - ../default
+  - ../scorecard
+  - ../samples
```

Comment:
This is translated to the `alm-examples` annotation when the `bundle` directory is generated by `make bundle`.